treeherder/etl/pulse_consumer.py

```diff
@@ -6,7 +6,7 @@
 from kombu.mixins import ConsumerMixin
 
 from treeherder.etl.common import fetch_json
-from treeherder.etl.tasks.pulse_tasks import (store_pulse_jobs,
```

Comment:
Let's actually leave this task as ``store_pulse_jobs``.  This is a celery task, and plural here is ok.  Plus, we have existing queues that would have to be drained and so it would require some crossover code.  I don't think it's worth it at this time.